tools/moth.py

```diff
@@ -328,15 +334,35 @@ def __init__(self, path, seed):
         self.path = path
         self.seed = seed
         self.pointvals = []
+        self.catmod = None
+
+        try:
+            catmod = SourceFileLoader(
+                'catmod',
+                os.path.join(path, 'category.py')).load_module()
+            assert all([
+                hasattr(catmod, 'make'),
+                hasattr(catmod, 'points'),
+                type(catmod.points) is list,
+            ])
+            self.catmod = catmod
+            self.pointvals.extend(catmod.points)
```

Comment:
If you define a category module, it needs to generate the entire category. This prevents the discrepancy where the category generates a point value that's also got a directory.